Android/app/src/main/java/live/dittolive/chat/profile/ProfileFragment.kt

```diff
@@ -115,7 +117,14 @@ class ProfileFragment : Fragment() {
                 val userData by profileViewModel.userData.observeAsState()
                 val nestedScrollInteropConnection = rememberNestedScrollInteropConnection()
                 val isEditMode = profileViewModel.isEditMode.collectAsStateWithLifecycle()
+                val shouldCloseProfile = profileViewModel.shouldCloseProfile.collectAsStateWithLifecycle()
 
+                LaunchedEffect(shouldCloseProfile.value) {
+                    if (shouldCloseProfile.value) {
+                        profileViewModel.resetCloseSignal()
+                        activity?.onBackPressed()
```

Comment:
`onBackPressed()` is deprecated

```suggestion
                        activity?.onBackPressedDispatcher?.onBackPressed()
```